apis/metal3.io/v1alpha1/baremetalhost_types.go

```diff
@@ -722,12 +741,14 @@ type BareMetalHostStatus struct {
 	// Important: Run "make generate manifests" to regenerate code
 	// after modifying this file
 
-	// OperationalStatus holds the status of the host
+	// OperationalStatus holds the status of the host and its
+	// validated by the checkStatusAnnotation function
```

Comment:
Same problem: the consumers of the API have no idea about functions in the Go code.

Reply:
Done